ui/lib/pki/addon/components/pki-page-header.hbs

```diff
@@ -0,0 +1,36 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb" aria-label="breadcrumbs">
+      <ul>
+        <li>
+          <span class="sep">/</span>
+          <LinkToExternal @route="secrets">secrets</LinkToExternal>
+        </li>
+        <li>
+          <span class="sep">/</span>
+          <span>{{@backend.id}}</span>
+        </li>
+      </ul>
+    </nav>
```

Comment:
I wonder if it's worth using the `<Page::Breadcrumbs>` component from `core` here? Not sure if that would make it easier or not when moving to HDS. Also, it seems like there is a crumb missing for the current route (overview, roles etc.).

Reply:
Ahh yes. I was originally thinking about some of the child routes like configure for example but it looks like they all have their own page headers with breadcrumbs. Confused myself 😵‍💫!